compiler/modules/front/resolveTypes.bal

```diff
@@ -329,6 +338,18 @@ function resolveTypeDesc(ModuleSymbols mod, s:ModuleLevelDefn modDefn, int depth
     panic error("unimplemented type-descriptor");
 }
 
+function nonEmptyType(ModuleSymbols mod, s:ModuleLevelDefn modDefn, s:TypeDesc td, t:SemType ty) returns t:SemType|ResolveTypeError {
+    final t:Env env = mod.tc.env;
```

Comment:
Not much point to this variable since we are using it only once.

Reply:
Fixed